### ISSUE TITLE
Allow circle as alternative shape for centromere, different color?

### ISSUE THREAD

**Issue by ivanek** [ivanek/Gviz#7]
Based on user request: __How to modify centromere depiction in Gviz IdeogramTrack?:__
[https://support.bioconductor.org/p/123093/](https://support.bioconductor.org/p/123093/)

```
library(Gviz)

cyto.bands <- data.frame(chrom = rep("chrI", 4),
                         chromStart = c(0, 148071, 151524, 154977),
                         chromEnd = c(148071, 151524, 154977, 230218),
                         name = c(NA, "CEN1", "CEN1", NA),
                         gieStain = c("gneg", "acen", "acen", "gneg"))

ideogram.track <- IdeogramTrack(chromosome = "chrI", genome = "sacCer3", bands = cyto.bands)

plotTracks(ideogram.track, from = 180e3, to = 220e3)
```

In the resulting plot, the centromere is depicted as two head-to-head triangles with brownish red filling. Is it possible to change how the centromere is depicted? Can one change the fill color? Can one change the shape (e.g. one filled circle instead of the triangles)?

**Comment by ivanek** [comment]
The color can be changed via `biovizBase` package by changing the defaults

```
opts <- getOption("biovizBase")
opts$cytobandColor["acen"] <- "red"
options(biovizBase = opts)
```

and reverting it back by calling

```
opts$cytobandColor <- biovizBase::getBioColor("CYTOBAND", source = "default")
options(biovizBase = opts)
```

**Comment by ivanek** [comment]
Fixed in 1.31.8.
